### PR TITLE
Confirm before deleting application

### PR DESCRIPTION
**Pull request by containscafeine**
This commit prompts the user before deleting the application when
`ocdev application delete name` command is run.

Also, a "--force/-f" flag has been introduced which skips this
prompt if set.

### CODE REVIEW

**Comment by kadel** [comment]
Can you please link this PR to the issue?

File: cmd/application.go [hunk 1]
```diff
@@ -78,12 +80,27 @@ var applicationDeleteCmd = &cobra.Command{
...
+		if strings.ToLower(confirmDeletion) == "y" {
+			err := application.Delete(appName)
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+			fmt.Printf("Deleted application: %s\n", args[0])
+		} else {
+			fmt.Printf("Aborting deletion of application: %v\n", appName)
+			os.Exit(0)
```
Comment (kadel) [inline cmd/application.go]: don't use exit here if you are not returning error code, just `return`. Or in this case, there doesn't have to be anything

Comment (containscafeine) [reply cmd/application.go]: fixed

**Comment by containscafeine** [comment]
> Can you please link this PR to the issue?

The issue is for component, I'll send a PR for it now

**Comment by kadel** [comment]
> The issue is for component, I'll send a PR for it now

what about this one? https://github.com/redhat-developer/ocdev/issues/203

also related to related to https://github.com/redhat-developer/ocdev/issues/202

**Comment by containscafeine** [comment]
missed that.
I've made the changes, can we merge this?

**Pull request merged**